libs/estdlib/src/io_lib.erl

```diff
@@ -292,6 +292,8 @@ format_spw(_Format, T) when is_float(T) ->
     erlang:float_to_list(T);
 format_spw(_Format, T) when is_pid(T) ->
     erlang:pid_to_list(T);
+format_spw(_Format, T) when is_port(T) ->
+    erlang:port_to_list(T);
```

Comment:
Thank you so much.
Could we have a test for this clause that passes on both BEAM and AtomVM?

`open_port({spawn, "echo"}, [])` works on both platforms AFAIK.

Reply:
Absolutely, I should have thought to add tests... "echo" is what I have been using to test locally ;-)

Reply:
> Thank you so much. Could we have a test for this clause that passes on both BEAM and AtomVM?
> 
> `open_port({spawn, "echo"}, [])` works on both platforms AFAIK.

Reply:
> Thank you so much. Could we have a test for this clause that passes on both BEAM and AtomVM?


The short answer is no. Lol! Any test that passes on the BEAM will fail on AtomVM until #1473 is merged. But I will write one that will make sure we are returning a list that matches the format that OTP returns.

Reply:
We need to make 1474 a continuation of 1473 :-)